pkg/common/args.go

```diff
@@ -41,7 +48,7 @@ func PrintInfo(clientSet *kubernetes.Clientset, config *rest.Config) {
 		viper.Set("busybox-image", busyboxImage)
 	}
 
-	log.Printf("API endpoint : %s", config.Host)
+	log.PrintfAPI("API endpoint : %s", config.Host)
```

Comment:
Hey @rjsadow I made a new method ` PrintAPI`  because I wanted the Spinner to be at top most and below this the function to be running and when using `Printf` the output is-
![Screenshot from 2024-02-11 23-41-05](https://github.com/kubernetes-sigs/hydrophone/assets/108119109/08a14fd0-9488-4c7c-a2db-c74e22a99a75)

and when i create separate method for API line the output is-
![Screenshot from 2024-02-11 23-41-29](https://github.com/kubernetes-sigs/hydrophone/assets/108119109/8ab78c27-d385-4216-9563-91bc437c7043)

My end goal is to keep the spinner at top most and everything below it